doc/xrcvd/transceiver-monitor-hld.md

```diff
@@ -284,9 +334,103 @@ Another entPhySensorTable which is defined in [Entity Sensor MIB(RFC3433)](https
 | 1.3.6.1.2.1.99.1.1.1.4.index | entPhySensorValue | Same as above | 7998 |
 | 1.3.6.1.2.1.47.1.1.1.1.2.index | entPhysicalDescr | Show interfaces alias | DOM RX Power Sensor for DOM RX Power Sensor for Ethernet29/1 |
 
-
 More detailed information about new table and new OIDs are described in [Sensor and Transceiver Info Monitoring Requirement](https://github.com/Azure/SONiC/blob/master/doc/xrcvd/OIDsforSensorandTransciver.MD#transceiver-requirements-entity-mib).
 
-
-
-      
+## 3. CLI change ##
+
+### 3.1 Add a command to fetch the error status ###
+
+#### 3.1.1 sfputil extension ####
+
+The utility `sfputil` needs to be updated to extend to support fetching error status of SFP modules.
+The CLI is like this:
+
+    sfputil show error-status --fetch-from-hardware --port <port>

```

Comment:
So without the `--fetch-from-hardware` flag, it will retrieve status from the `TRANSCEIVER_STATUS` table in `STATE_DB`. The reason for the `--fetch-from-hardware` flag is to ensure you're reading up-to-date data. Is that correct? Is there anything else special about this flag?

Reply:
Correct.
With `--fetch-from-hardware` provided, it will call platform API directly from pmon docker.